tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -49,29 +85,93 @@ describe('Header', () => {
     expect(menuBtn).toExist()
   })
 
-  it("should add usa-current class to Welcome tab when on '/edit-profile'", () => {
-    const store = mockStore(initialState)
+  it("should add usa-current class to Welcome tab when on '/welcome'", () => {
+    const store = mockStore({
+      ...initialState,
+      router: { location: { pathname: '/welcome' } },
+    })
     const wrapper = mount(
       <Provider store={store}>
         <Header />
       </Provider>
     )
 
-    const welcomeTab = wrapper.find('.usa-nav__link')
+    const welcomeTab = wrapper.find('#welcome')
 
     expect(welcomeTab.hasClass('usa-current')).toEqual(true)
   })
 
+  it("should add usa-current class to Reports tab when on '/reports'", () => {
+    const store = mockStore({
+      ...initialState,
+      router: { location: { pathname: '/reports' } },
+    })
+    const wrapper = mount(
+      <Provider store={store}>
+        <Header />
+      </Provider>
+    )
+
+    const reportsTab = wrapper.find('#reports')
+
+    expect(reportsTab.hasClass('usa-current')).toEqual(true)
+  })
+
+  it("should add usa-current class to Reports tab when on '/reports/*'", () => {
```

Comment:
Ensure that the `Reports` navigation tab is still active when on a subroute.

Reply:
One request for a future story/release is when the `usa-current` class is added to primary nav item for the page, also add the ARIA attribute and value of `aria-current="page"` to the anchor element. This will supplement the experience for screen reader users who cannot see the visual current item indicator that folks who can see will receive. 

Here's a [very good article from Léonie Watson's blog about using `aria-current`](https://tink.uk/using-the-aria-current-attribute/).